### PR TITLE
Fixed payment binding

### PR DESCRIPTION
**Pull request by wulfthimm**
## Description

The secret containing the payment credentials can not be copied to a preview-environment.
Symptom:
```
- [x] with-helm=true
- [x] with-payment=true
fails with:
cp: cannot stat '/mnt/secrets/payment-webhook-config/webhook': No such file or directory
[deploy|FAIL] Error: cp /mnt/secrets/payment-webhook-config/webhook payment-core-dev-webhook.json exit with non-zero status code
```
See https://werft.gitpod-dev.com/job/gitpod-build-jx-better-team-subscriptions.3/raw

This solves the problem by using the correct filename.

## Related Issue(s)

Fixes #

## How to test

Either check https://werft.gitpod-dev.com/job/gitpod-build-wth-fix-payment.0/raw or push this commit to a new branch and check if the payment endpoint is working correctly.

## Release Notes

```release-note
None
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **felladrin** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @felladrin` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jankeromnes — APPROVED** [review]
Many thanks for the quick fix @wulfthimm!

I could not find where the file names `webhook` or `license` come from in the `payment-webhook-config` secret, but since your build with identical options is green, this seems correct! 🎯

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 187e99ef4b5418e15d83c2456f0a899353929b8a</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/8057#pullrequestreview-874306596" title="Approved">jankeromnes</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/8057#issuecomment-1031181733" title="Approved">jankeromnes</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/8057#issuecomment-1031181733" title="Approved">jankeromnes</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
